Mathlib/Topology/Algebra/Algebra/Equiv.lean

```diff
@@ -0,0 +1,296 @@
+/-
+Copyright (c) 2024 Salvatore Mercuri. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Salvatore Mercuri
+-/
+import Mathlib.Topology.Algebra.Algebra
+
+/-!
+# Isomorphisms of topological algebras
+
+This file contains an API for `ContinuousAlgEquiv R A B`, the type of
+continuous `R`-algebra isomorphisms with continuous inverses. Here `R` is a
+commutative (semi)ring, and `A` and `B` are `R`-algebras with topologies.
+
+## Main definitions
+
+Let `R` be a commutative semiring and let `A` and `B` be `R`-algebras which
+are also topological spaces.
+
+* `ContinuousAlgEquiv R A B`: the type of continuous `R`-algebra isomorphisms
+  from `A` to `B` with continuous inverses.
+
+## Notations
+
+`A ≃A[R] B` : notation for `ContinuousAlgEquiv R A B`.
+
+## Tags
+
+* continuous, isomorphism, algebra
+-/
+
+open scoped Topology
+
+
+/--
+`ContinuousAlgEquiv R A B`, with notation `A ≃A[R] B`, is the type of bijections
+between the topological `R`-algebras `A` and `B` which are both homeomorphisms
+and `R`-algebra isomorphisms.
+-/
+structure ContinuousAlgEquiv (R A B : Type*) [CommSemiring R]
+    [Semiring A] [TopologicalSpace A] [Semiring B] [TopologicalSpace B] [Algebra R A]
+    [Algebra R B] extends A ≃ₐ[R] B, A ≃ₜ B where
```

Comment:
The `where` is not necessary when there are no new fields, and I seem to remember that the preferred style is to leave it out (not 100% sure, though). Same below. Other than that, this looks OK to me.

Reply:
Thanks!